tests/middleware/test_message_state.py

```diff
@@ -0,0 +1,34 @@
+import remoulade
+from remoulade.state.message_state import MessageState
+
+
+class TestMessageState:
+    """Class to test the middleware
```

Comment:
You are missing the stub state backend. It's very usefull for testing. Could you add it and do test on both ?